docs/src/inputs/radio/api/api.jsx

```diff
@@ -0,0 +1,38 @@
+import {
+    camelCase,
+} from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ComponentApi from '../../../global/componentApi';
+import Main from '../../../global/main';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as rootDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/radio/radio';
+/* eslint-enable import/no-named-default, import/extensions */
+
+const propTypes = {
+    location: PropTypes.shape({
+        pathname: PropTypes.string,
+    }).isRequired,
+};
```

Comment:
Please remove `propTypes`. It is not being used.

Reply:
Done!